dapps/assets/src/components/contacts/EditContact.vue

```diff
@@ -0,0 +1,102 @@
+/*
+  Copyright (C) 2018-present evan GmbH.
+
+  This program is free software: you can redistribute it and/or modify it
+  under the terms of the GNU Affero General Public License, version 3,
+  as published by the Free Software Foundation.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+  See the GNU Affero General Public License for more details.
+
+  You should have received a copy of the GNU Affero General Public License
+  along with this program. If not, see http://www.gnu.org/licenses/ or
+  write to the Free Software Foundation, Inc., 51 Franklin Street,
+  Fifth Floor, Boston, MA, 02110-1301 USA, or download the license from
+  the following URL: https://evan.network/license/
+*/
+
+<template>
+  <form @submit.prevent="onSubmit">
+    <evan-swipe-panel
+      ref="editContactPanel"
+      alignment="right"
+      type="default"
+      class="light"
+      :show-backdrop="true"
+      :title="$t('_assets.contacts.edit-contact-title')"
+    >
+      <evan-loading v-if="!contact" />
+
+      <template v-else>
+        <evan-form-control-input
+          :value="contact.address"
+          :disabled="true"
+          :required="true"
```

Comment:
required true seems bit redundant, when it's disabled

Reply:
ah never mind, it's the optional label right?

Reply:
Yeah, it's just there for the label